src/nxdoc/getting-started/best-practices/choosing-how-to-import-data-in-the-nuxeo-platform.md

```diff
@@ -169,58 +172,71 @@ May it be a file system binary store, an S3 binary store or an Azure Object stor
 
 ## Existing Import Tools
 
-### node.js Importer
+### Nuxeo Platform Importer
 
-**Features**
+The Nuxeo Platform Importer comes with several implementations:
+- **Nuxeo Bulk importer**, the generic mass document import addon (full, multi-threaded import from the server file system).
+- **Nuxeo Scan importer** (to create documents from XML files).
+- **Nuxeo Stream importer** (to create documents with a producer/consumer pattern logic, and uses the Log features provided by [Nuxeo Stream]({{{page space='nxdoc' page='nuxeo-stream'}}).
 
-The [node.js importer](https://github.com/nuxeo-sandbox/nuxeo-node-importer) makes use of the REST API and provides you with additional services compared to the bare approach:
+#### Nuxeo Bulk Importer
 
-- Multi-threading
-- Client-side browsing of a complete hierarchy of content (folders, subfolders and files)
+**Features**
+
+The [Nuxeo Bulk Document Importer](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-platform-importer) is an importer framework provided as an addon that can be used to build custom importers. It relies on a standard crawler, transformer, writer schema. The Scan importer and CSV importer addons are using that framework (see next sections). It is the de-facto choice when you want to reach hyperscale numbers with importing content (up to 10&nbsp;000s of documents per second). All you need to do is write your own Document Factory that will be in charge of the document creation logic in the repository. You can then easily launch the import controlling how many documents are done in a batch, how many batches per transaction, etc.
```

Comment:
It is the de-facto choice ... All you need to do is write your own ->
This is a solution for mass import that can handle thousands of documents per second. You need to write your own Document Factory ...